wikibase/README.md

```diff
@@ -9,10 +9,16 @@ Automated build.
 
 ### Tags
 
-Image name                                  | Parent image            
-------------------------------------------- | ------------------------
-`wikibase/wikibase` : `latest`, `1.30`      | [mediawiki:1.30](https://hub.docker.com/_/mediawiki/)
-`wikibase/wikibase` : `legacy`, `1.29`      | [mediawiki:1.29](https://hub.docker.com/_/mediawiki/)
+Image name                                                       | Parent image
+---------------------------------------------------------------- | ------------------------
+`wikibase/wikibase` : `latest`, `1.30`                           | [mediawiki:1.30](https://hub.docker.com/_/mediawiki/)
+`wikibase/wikibase-base` : `latest`, `1.30` (identical to above) | [mediawiki:1.30](https://hub.docker.com/_/mediawiki/)
```

Comment:
This is actually wrong, -base is part of the tag name, not part of the image name. Same for -bundle.

Reply:
Cool, fixing